src/design-system-team.md

```diff
@@ -33,4 +33,5 @@ If you want to contact the team you can [get in touch via email or Slack](/get-i
 - Kim 'beeps' Grey – Frontend Developer
 - Mia Allers – Senior Graphic Designer
 - Owen Jones – Senior Frontend Developer
+- Patrick Cartlidge - Frontend Developer
```

Comment:
A minor thing – for consistency can we make this an em dash rather than a hyphen?

```suggestion
- Patrick Cartlidge – Frontend Developer
```

You can see the difference on [the preview](https://deploy-preview-3733--govuk-design-system-preview.netlify.app/design-system-team/):

<img width="426" alt="Screenshot of the team page with mismatched separators" src="https://github.com/alphagov/govuk-design-system/assets/121939/f706d728-9755-433e-a23a-ab33e7556852">

 There's a note about this on line 7 but it's easy to miss. I promise it's not a test! 😆

Reply:
Ah, sorry about that! In my editor the hyphen and em dash looked the same!

Reply:
This catches out literally everyone. It's basically a rite of passage. 😝